### PR TITLE
Int 8284 - Add ingestion sources

### PR DESCRIPTION
**Pull request by VDubber**

### CODE REVIEW

File: docs/jupiterone.md [hunk 1]
```diff
@@ -162,7 +162,6 @@ The following entities are created:
 | GitHub Environment             | `github_environment`            | `Configuration` |
 | GitHub Issue                   | `github_issue`                  | `Issue`         |
 | GitHub Org Secret              | `github_org_secret`             | `Secret`        |
-| GitHub Outside Collaborator    | `github_user`                   | `User`          |
```
Comment (VDubber) [inline docs/jupiterone.md:165]: revert

Comment (dobregons) [reply docs/jupiterone.md:165]: Out of curiosity, why this was removed?

**Review by dobregons — APPROVED** [review]
LGTM!

File: docs/jupiterone.md (hunk 1 above)
Comment (VDubber) [reply docs/jupiterone.md:165]: I think there is a bug in the documentation generator. There's another github_user entity row and it probably dedupped

**Review by dobregons — APPROVED** [review]

**Pull request merged**